packages/store/src/ResourceId.sol

```diff
@@ -3,15 +3,14 @@ pragma solidity >=0.8.21;
 
 type ResourceId is bytes32;
 
-uint256 constant TYPE_BYTES = 2;
-uint256 constant NAME_BYTES = 32 - TYPE_BYTES;
-uint256 constant BYTES_TO_BITS = 8;
+uint256 constant TYPE_BITS = 2 * 8;
+uint256 constant NAME_BITS = 256 - TYPE_BITS;
```

Comment:
```suggestion
uint256 constant NAME_BITS = 32 * 8 - TYPE_BITS;
```
tiny bit clearer

Reply:
also totally a nit, feel free to ignore, but I like to put the smallest unit on the left

like when I do time calculations, I usually always aim for
```
1000 * 60 * 60 * 24
```